recipes/geotrans/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,29 @@
+from conans import ConanFile, CMake, tools
+from contextlib import contextmanager
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    @contextmanager
+    def _workaround_2profiles(self):
```

Comment:
🤔 what is this workarounding?

Reply:
`self.env_info` of requirements are not injected when calling `self.run(..., run_environment=True)` when using 2 profiles.